tests/playbooks/integration_pytest_python3.yml

```
@@ -0,0 +1,104 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Install dependencies for integration tests
+  hosts: all
+  vars:
+    - rpmdependencies:
+        - git
+        - python3-pip
+        - rsync
+
+  tasks:
+    - name: Install EPEL for RHEL and CentOS
+      # yamllint disable-line rule:line-length
+      command: "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
+      when: ansible_distribution in ["CentOS", "RedHat"]
+
+    - name: Install rpm dependencies
+      package:
+        state: present
+        name: "{{ rpmdependencies }}"
+
+    - name: Search for rsync path
+      command: which rsync
+
+    - name: Install Pytest
+      command: "pip3 install pytest"
+
+
+# Import needed in order to install initscripts dependencies on the remote
+# system.
+- import_playbook: "../tests_default_initscripts.yml"
+- hosts: all
+
+# Import needed in order to install Network Manager dependencies on the remote
+# system.
+- import_playbook: "../tests_default_nm.yml"
+- hosts: all
+
+
+- name: Run Pytest tests
+  hosts: all
+  vars:
+    - rundir: /run/system-roles-test/
+  tasks:
+    - file:
+        state: directory
+        path: /run
```

```suggestion
        path: "{{ rundir }}"
        recurse: true
```

If this works, it would allow to avoid the hardcoded `/run` directory here and only use the variable.

---

It works!! Thanks